.gitignore

```diff
@@ -9,6 +9,7 @@ __pycache__
 external
 .idea/
 .vscode/
+.DS_Store
```

Comment:
FYI, it might be helpful to set a global gitignore for stuff like this so you don't have to do that on every project, something like this if you have a `.gitignore` file in your home directory.
`git config --global core.excludesFile '~/.gitignore'`